ludwig/schema/config/utils.py

```diff
@@ -0,0 +1,74 @@
+import yaml
+
+from ludwig.constants import ACTIVE, COLUMN, INPUT_FEATURES, NAME, OUTPUT_FEATURES, PROC_COLUMN, TYPE
+from ludwig.features.feature_utils import compute_feature_hash
+from ludwig.schema.utils import convert_submodules
+from ludwig.utils.registry import Registry
+
+internal_output_config_registry = Registry()
+
+
+def get_output_metadata_cls(name: str):
+    return internal_output_config_registry[name]
+
+
```

Comment:
All of the below functions are things that have already been reviewed, they're just moved into this util file for common use and better organization.

Reply:
BaseFeaturesContainer, InputFeaturesContainer, OutputFeaturesContainer could live in their own python module.   `features_container.py` or something like that.

Reply:
Sure I can do that. Before I make the change however, is there a certain reason why we don't want them in utils.py here?